Dockerfile

```diff
@@ -22,11 +22,11 @@ RUN apk add --no-cache \
 
 RUN npm config set unsafe-perm true
 RUN npm install -g \
-        --registry http://registry.npmjs.org/ \
-        coffee-script \
+        --registry https://registry.npmjs.org/ \
+        coffeescript \
         less@1.3
 
-RUN pip install --upgrade pip
+RUN pip install --upgrade pip==20.3.4
```

Comment:
The related commit messages says "20.4.5" but the code says "20.3.4" :thinking: :smiley:  How about `pip<21`? Gave me 20.3.4 in practice over here.